WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -64,7 +64,8 @@ public static int Main(string[] args)
 					SetupLogger(dataDir, args);
 					var (uiConfig, config) = LoadOrCreateConfigs(dataDir);
 
-					singleInstanceChecker = new SingleInstanceChecker(config.Network);
+					using var sic = new SingleInstanceChecker(config.Network);
```

Comment:
This is not correct. We need SingleInstanceChecker outside of scope.

Reply:
Reverted